src/effects/VST3/VST3Effect.cpp

```diff
@@ -0,0 +1,787 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file VST3Effect.cpp
+
+  @author Vitaly Sverchinsky
+
+  @brief Part of Audacity VST3 module
+
+**********************************************************************/
+
+#include "VST3Effect.h"
+
+#include <stdexcept>
+#include <wx/log.h>
+#include <pluginterfaces/vst/ivstaudioprocessor.h>
+#include <pluginterfaces/vst/ivsteditcontroller.h>
+#include <pluginterfaces/vst/ivstprocesscontext.h>
+#include <wx/msw/spinctrl.h>
```

Comment:
The build is broken if not on Windows.  That must be fixed of course.  One if the fixes is simply to omit "/msw" in that path.  There are other fixes to do in VST3Utils.cpp